gate-keeper/src/main/java/com/iluwatar/gatekeeper/LoginRequst.java

```diff
@@ -0,0 +1,21 @@
+package com.iluwatar.gatekeeper;
+
+import lombok.Getter;
+
+@Getter
+public class LoginRequst extends Request {
```

Comment:
```suggestion
public class LoginRequest extends Request {
```

Reply:
I have change it， and I change the image~